huaweicloud/services/cbr/resource_huaweicloud_cbr_vault.go

```diff
@@ -399,18 +432,40 @@ func resourceCBRVaultV3Read(d *schema.ResourceData, meta interface{}) error {
 }
 
 func updateResources(d *schema.ResourceData, client *golangsdk.ServiceClient) error {
-	resources, err := buildCBRVaultResources(d.Get("type").(string), d.Get("resources").(*schema.Set))
-	if err != nil {
-		return fmtp.Errorf("Error building vault resources: %s", err)
-	}
+	old, new := d.GetChange("resources")
+	addRaws := new.(*schema.Set).Difference(old.(*schema.Set))
+	delRaws := old.(*schema.Set).Difference(new.(*schema.Set))
 
```

Comment:
variable `new` collides with the builtin function

Reply:
fixed